its/projects/IgnoreIssuesDoesNotRemoveSourceGenerator/global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "5.0.0",
```

Comment:
can this be net 6 or we don't have the infra yet?

Reply:
ITs run with both MsBuild16 and 17, and it would not be possible to build the solution with MSBuild16.